xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -792,9 +792,14 @@ bool CDVDInputStreamNavigator::OnMouseClick(const CPoint &point)
   return false;
 }
 
-void CDVDInputStreamNavigator::OnMenu()
+bool CDVDInputStreamNavigator::OnMenu()
 {
-  if (m_dvdnav) m_dll.dvdnav_menu_call(m_dvdnav, DVD_MENU_Escape);
+  if (!m_dvdnav)
+  {
+    return false;
+  }
+
+  return m_dll.dvdnav_menu_call(m_dvdnav, DVD_MENU_Escape) == DVDNAV_STATUS_OK;
```

Comment:
i think the menu is definitely called at this point. should we just return true regardless of the status?

Reply:
dvdnav_menu_call may return DVDNAV_STATUS_ERR and in this case the menu was not opened successfully, and we return false. I don't really got your question/point, see https://github.com/xbmc/libdvdnav/blob/981488f7f27554b103cca10c1fbeba027396c94a/src/searching.c#L591

Reply:
you‘re right i didn‘t see there‘s only two states OK and ERR. all good